tutorial/chapter/3/index.md

```diff
@@ -173,11 +184,12 @@ fn foo(x) x + 1
 This is useful for functions that need to refer to themselves, like:
 
 ```squiggle
-fn forever(f)
-    if done
-    then undefined
+def forever(f)
```

Comment:
i guess what I mean is, it should simply be
```
let forever = fn (f)  do
```
without naming it twice.
that way do end becomes a block, and clearly so, and it's no longer necessary to duplicate the name, and `def` becomes redundant.

i think this would make it more consistent, and closer in spirit to js